share/spack/gitlab/cloud_pipelines/configs/ci.yaml

```diff
@@ -110,6 +110,8 @@ ci:
         CI_JOB_SIZE: "small"
         KUBERNETES_CPU_REQUEST: "500m"
         KUBERNETES_MEMORY_REQUEST: "500M"
+      script:
+      - exit 1
```

Comment:
Is there a reason we have the no-op job always return a failing code?

Although we allow failures it might be nice to return 0 normally so we can filter (via API) later on to find ones that really end up in a failed state.

Reply:
This is a fix up, I wanted to make sure failure still passes since we don't run CI in the unit-tests. I am going to squash/remove it